### PR TITLE
Solana environment WIP :construction: 

### PR DESCRIPTION
**Pull request by albert-llimos**
# Pull Request

Closes: PRO-xxx

## Checklist

Please conduct a thorough self-review before opening the PR.

- [ ] I am confident that the code works.
- [ ] I have updated documentation where appropriate.

## Summary

*Please include a succinct description of the purpose and content of the PR. What problem does it solve, and how? Link issues, discussions, other PRs, and anything else that will help the reviewer.*

#### Non-Breaking changes

If this PR includes non-breaking changes, select the `non-breaking` label. On merge, CI will automatically cherry-pick the commit to a PR against the release branch.

### CODE REVIEW

File: .github/workflows/ci-development.yml [hunk 1]
```diff
@@ -1,10 +1,7 @@
 # This CI runs a lot of the jobs in parallel to speed up development time. We also run a simpler suite of bouncer tests.
 name: Release Chainflip Development
-on:
-  pull_request:
-    branches:
-      - main
-      - "release/*"
+on: push
```
Comment (syan095) [inline .github/workflows/ci-development.yml:3]: Is it Ok to change CI here?

File: state-chain/pallets/cf-environment/src/lib.rs [hunk 2]
```diff
@@ -201,10 +203,55 @@ pub mod pallet {
 	pub type ArbitrumSignatureNonce<T> = StorageValue<_, SignatureNonce, ValueQuery>;
 
 	// SOLANA CHAIN RELATED ENVIRONMENT ITEMS
+	#[pallet::storage]
+	#[pallet::getter(fn supported_sol_assets)]
+	/// Map of supported assets for SOL
+	pub type SolanaSupportedAssets<T: Config> =
```
[the remark this answers is not in the document]
Comment (albert-llimos) [reply state-chain/pallets/cf-environment/src/lib.rs:209]: yes you're right, I'll do that.

[the remark this answers is not in the document]
Comment (dandanlen) [reply state-chain/pallets/cf-environment/src/lib.rs:209]: (Or a big struct.)

**Comment by syan095** [comment]
This PR is now been replaced by  #5016

**Pull request closed without merging**